src/symbolic/grid_based_symbolic_model/grid_based_symbolic_model.jl

```diff
@@ -0,0 +1,333 @@
+using Plots, Colors
```

Comment:
```suggestion
import Plots, Colors
```

Reply:
I've made this clear so that we can easily consider removing Plots from Dionysos later. It is still used for really few (not really usefull functions).